recipe/meta.yaml

```diff
@@ -25,7 +24,7 @@ requirements:
     - python
     - parse >=1.6
     - six
-    - enum34
+    - enum34  # [py2k]
```

Comment:
No, what exactly is todo? I just did a python36 build on windows, and it worked. 
Where is described what needs to be done? Does it work with windows?

I experimented with "conda smithy rerender" but that failed, deleting .git files